nautobot/core/api/serializers.py

```diff
@@ -159,7 +159,7 @@ def to_representation(self, value):
             model = self.queryset.model
 
         # foreign key relationship when the destination field is referenced by string not by class
-        elif getattr(self.parent.Meta.model, self.source, False):
+        elif getattr(self.parent, "Meta", None) and getattr(self.parent.Meta.model, self.source, False):
```

Comment:
Is this needed after #3916? It looks like a reasonable change, but is it fixing an issue that is actually encountered?

Reply:
Nope